src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -6885,8 +6887,17 @@ static void sofia_handle_sip_r_invite(switch_core_session_t *session, int status
 		if (status >= 400 && sip->sip_reason && sip->sip_reason->re_protocol && (!strcasecmp(sip->sip_reason->re_protocol, "Q.850")
 				|| !strcasecmp(sip->sip_reason->re_protocol, "FreeSWITCH")
 				|| !strcasecmp(sip->sip_reason->re_protocol, profile->sdp_username)) && sip->sip_reason->re_cause) {
+			char status_str[5];
+			const char * session_ignore_list = switch_channel_get_variable(channel, "ignore_reason_header_by_sip_code");
+			const char * current_ignore_list = !zstr(session_ignore_list) ? session_ignore_list : profile->ignore_reason_header_by_sip_code;
```

Comment:
For consistency's sake with all the rest of the core FS code, can we instead do:

const char *session_ignore_list = switch_channel_get_variable(channel, "ignore_reason_header_by_sip_code");
const char *current_ignore_list = !zstr(session_ignore_list) ? session_ignore_list : profile->ignore_reason_header_by_sip_code;